reviews/module.py

```diff
@@ -0,0 +1,314 @@
+    @commands.guild_only()
+    @review.command(name="add-anonymous", aliases=["anonymous", "anon"])
+    async def review_add_anonymous(
+        self, ctx: discord.ext.commands.Context, subject: str, mark: int, *, text: str
+    ):
+        """Adds an anonymous review and deletes your message.
+
+        subject: Subject code
+        mark: 1-5 (one being best)
+        text: Your review
+        """
+        result = await self._add_review(ctx, subject, mark, text, True)
+        if result is not None:
+            await guild_log.info(
+                ctx.author,
+                ctx.channel,
+                f"Added anonymous review for subject {subject}.",
+            )
+            await ctx.send(_(ctx, "Anonymous review successfully added."))
+        await ctx.message.delete()
+
+    @check.acl2(check.ACLevel.MEMBER)
+    @commands.guild_only()
+    @review.command(name="remove", aliases=["delete"])
+    async def review_remove(self, ctx: discord.ext.commands.Context, subject: str):
+        """Remove your review
+
+        subject: Subject abbreviation
+        """
+        if Review.remove(ctx.guild, ctx.author, subject):
+            await guild_log.info(
+                ctx.author, ctx.channel, f"Removed their review for subject {subject}."
+            )
+            await ctx.reply(_(ctx, "Review deleted successfully."))
+            return
+        await ctx.reply(_(ctx, "Review for this subject not found."))
+
+    @check.acl2(check.ACLevel.MEMBER)
+    @commands.guild_only()
+    @commands.group(name="subject")
+    async def subject(self, ctx):
+        """Manage subjects."""
+        await utils.discord.send_help(ctx)
+
+    @check.acl2(check.ACLevel.MEMBER)
+    @commands.guild_only()
+    @subject.command(name="info")
+    async def subject_info(self, ctx: discord.ext.commands.Context, subject: str):
+        """Get information about subject
+
+        subject: Subject code
+        """
+        subject = Subject.get(ctx.guild, subject)
+        if subject is None:
+            return await ctx.reply(_(ctx, "Unknown subject."))
+
+        embed = discord.Embed(title=subject.shortcut)
+        embed.add_field(name=_(ctx, "Name"), value=subject.name)
+        embed.add_field(name=_(ctx, "Department"), value=subject.category)
+        embed.add_field(
+            name=_(ctx, "Number of reviews"), value=len(list(subject.reviews))
+        )
+
+        await ctx.reply(embed=embed)
+
+    @check.acl2(check.ACLevel.MOD)
+    @commands.guild_only()
+    @subject.command(name="add", aliases=["update"])
+    async def subject_add(
+        self,
+        ctx: discord.ext.commands.Context,
+        abbreviation: str,
+        name: str,
+        category: str,
+    ):
+        Subject.add(ctx.guild, abbreviation, name, category)
+        await guild_log.info(
+            ctx.author, ctx.channel, f"Added/updated subject: {abbreviation}."
+        )
+        await ctx.reply(_(ctx, "Subject added."))
+
```

Comment:
The command is missing a detailed description that is shown with the use of `!help [command]` or with the use of the command without parameters.